### ISSUE TITLE
Pip installation fails

### ISSUE THREAD

**Issue by ekovalik** [databricks/databricks-cli#261]
I am using Python 3.7.4 (x64) on Windows 10, using the latest PIP package (19.2.3), When I try to install the Databricks CLI, it fails with the following error: 

```
2019-09-24T22:09:56,606   Using cached https://files.pythonhosted.org/packages/76/35/ae65ed1268d6e2a1be141723e5fffdf4a28e4f4e7c1e083709b308998f90/tabulate-0.8.4.tar.gz
2019-09-24T22:09:56,607   Downloading from URL https://files.pythonhosted.org/packages/76/35/ae65ed1268d6e2a1be141723e5fffdf4a28e4f4e7c1e083709b308998f90/tabulate-0.8.4.tar.gz#sha256=dde537a1c44e6d9d635b2489923513bef89fadb181819998cccdf2a99e652464 (from https://pypi.org/simple/tabulate/)
2019-09-24T22:09:56,744   Added tabulate>=0.7.7 from https://files.pythonhosted.org/packages/76/35/ae65ed1268d6e2a1be141723e5fffdf4a28e4f4e7c1e083709b308998f90/tabulate-0.8.4.tar.gz#sha256=dde537a1c44e6d9d635b2489923513bef89fadb181819998cccdf2a99e652464 (from databricks-cli) to build tracker 'C:\\Users\\EDDIEK~1\\AppData\\Local\\Temp\\pip-req-tracker-9_yz8kj2'
2019-09-24T22:09:56,745     Running setup.py (path:C:\Users\EDDIEK~1\AppData\Local\Temp\pip-install-5cb_2xw6\tabulate\setup.py) egg_info for package tabulate
2019-09-24T22:09:56,746     Running command python setup.py egg_info
2019-09-24T22:09:56,973     Traceback (most recent call last):
2019-09-24T22:09:56,974       File "<string>", line 1, in <module>
2019-09-24T22:09:56,975       File "C:\Users\EDDIEK~1\AppData\Local\Temp\pip-install-5cb_2xw6\tabulate\setup.py", line 25, in <module>
2019-09-24T22:09:56,975         f.write(LONG_DESCRIPTION)
2019-09-24T22:09:56,976       File "c:\program files\python37\lib\encodings\cp1252.py", line 19, in encode
2019-09-24T22:09:56,976         return codecs.charmap_encode(input,self.errors,encoding_table)[0]
2019-09-24T22:09:56,977     UnicodeEncodeError: 'charmap' codec can't encode characters in position 5907-5924: character maps to <undefined>
```

I was not able to find a workaround. Any ideas?

**Comment by MilanBerka** [comment]
same problem here

**Comment by dlstadther** [comment]
Core issue is a downstream dependency of `tabulate`. See its [Issue](https://bitbucket.org/astanin/python-tabulate/issues/182/tabulate-fails-to-install-on-windows). Something in their 0.8.4 release on 2019-09-24 caused this problem.

If you `pip install tabulate==0.8.3` prior to installing `databricks-cli`, it'll work.

**Comment by dlstadther** [comment]
Looks to be resolved by `tabulate==0.8.5`. I think this issue can be closed

**Comment by ekovalik** [comment]
Yes - it's working fine now. Closing.
